src/scenes/levelend-scene.ts

```diff
@@ -84,131 +71,81 @@ export class LevelEndScene {
     this.audioPlayer = new AudioPlayer();
     this.starCount = starCount;
     this.currentLevel = currentLevel;
-    this.images = {
-      backgroundImg: WIN_BG,
-      star1Img: PIN_STAR_1,
-      star2Img: PIN_STAR_2,
-      star3Img: PIN_STAR_3,
-      winBackgroundImg: DEFAULT_BACKGROUND_1,
-    };
-    loadImages(this.images, (images) => {
-      this.loadedImages = Object.assign({}, images);
-      this.imagesLoaded = true;
-      this.starAnimation();
-    });
-    this.addEventListener();
-    this.audioPlayer = new AudioPlayer();
-    this.setupBg();
+
     this.isLastLevel =
       this.currentLevel !==
         this.data.levels[this.data.levels.length - 1].levelMeta.levelNumber &&
       this.starCount >= 2;
-      this.monster = new Monster(
+    
+    this.monster = new Monster(
       this.canvas,
       monsterPhaseNumber,
-      () => {
-        //Temp fix - wrapping this in a function to avoid reference undefined of 'this'.
-        //To do - hook level-end-scene properly to game state service.
-        this.switchToReactionAnimation
-      }
+      this.switchToReactionAnimation
     );
+    this.showLevelEndScreen();  // Display the level end screen
+    this.addEventListener();
+    this.renderStarsHTML();
   }
 
-  private setupBg = async () => {
-    const { BG_GROUP_IMGS, draw } = loadDynamicBgAssets(
-      this.currentLevel,
-      BACKGROUND_ASSET_LIST
-    );
-    this.background = await createBackground(
-      this.context,
-      this.width,
-      this.height,
-      BG_GROUP_IMGS,
-      draw
-    );
-  };
+  showLevelEndScreen() {
+    
+
+    // Make the levelEnd element visible by setting display to block
+    if (this.levelEndElement) {
+      this.levelEndElement.style.display = 'block';
+    }
+
+    // Render the stars dynamically based on the star count
+    this.renderStarsHTML();
+  }
 
   switchToReactionAnimation = () => {
     if (this.starCount <= 1) {
       if (isDocumentVisible()) {
         this.audioPlayer.playAudio(AUDIO_LEVEL_LOSE);
       }
-      this.monster.changeToSpitAnimation();
+      // this.monster.changeToSpitAnimation();
```

Comment:
I think we can completely removed this since we are no longer using Canvas.

Reply:
this is commented to handle the interactive animation on the levelend screen, I will add comments for more clarity.